tests/test_infographic_api.py

```diff
@@ -0,0 +1,304 @@
+import pytest
```

Comment:
Can you please use black before pushing this file?

Reply:
Done